### PR TITLE
Milestone 1:  Finished VTube User-Interface

### PR DESCRIPTION
**Pull request by prg-04**
## In this Milestone, I have done the following step to set up a "VTube" repository:

- [ ] Create a new GitHub repo.

- [ ] Add a `.gitignore` file.

- [ ] Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config).

- [ ] Make sure that any files that are not relevant are ignored by git.

- [ ] Add a descriptive README file ....

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+      </div>
+    </div>
+    <!--Main-->
+    <div class="laContainer">
+      <div class="banner">
+        <h1>Coding Theatre</h1>
+        <p>&lt;Watch and Learn... /&gt;</p>
+      </div>
+
+      <div class="list-container">
```
Comment (prg-04) [inline index.html:140]: I think you should use the grid layout for the video playlist for a better user experience.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+          ><img
+            src="https://yt3.ggpht.com/ytc/AMLnZu-fB-c8gczS81r-VBWgzndcSZgd1zGA2QcYfACHMg=s176-c-k-c0x00ffffff-no-rj-mo"
+            alt="Traversy Media"
+          />
+          <p>Traversy Media</p></a
+        >
+      </div>
+    </div>
+    <!--Main-->
+    <div class="laContainer">
```
Comment (prg-04) [inline index.html:134]: Nice work. However, is think this section needs some adjustment. divs have  no semantic meaning, I think you should use semantic tags such as section, article to group elements together

**Review by prg-04 — COMMENTED** [review]
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by levy002 — APPROVED** [review]
## Hello @prg-04  :wave: 
### I'm Levy, your PR reviewer!!

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPREOVED :fire: :fire: 
## Highlights
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**